clients/client-web/README.md

```diff
@@ -96,7 +96,6 @@ require(['taskcluster-client-web'], ({ Queue }) => {
 </script>
 ```
 
-Note that this module can be imported in a Node.js environment without error, but is not guaranteed to work.
```

Comment:
This was something that the treeherder team had requested in order to support server-side testing.  Is it no longer needed?

Reply:
Requiring this file in the node environment seem to throw an error about `window` not being defined.The `pretest` package.json script was failing.

Reply:
Oh, that's no fun.  @camd is that going to be a problem?  (ref https://bugzilla.mozilla.org/show_bug.cgi?id=1523376)